InWorldz/InWorldz.VivoxVoice/VivoxVoiceModule.cs

```diff
@@ -237,7 +237,7 @@ public void AddRegion(Scene scene)
                     
                     string channelId;
 
-                    string sceneUUID  = m_forcedChannelName == String.Empty ? scene.RegionInfo.RegionID.ToString() : m_forcedChannelName;
+                    string sceneUUID  = String.IsNullOrEmpty(m_forcedChannelName) ? scene.RegionInfo.RegionID.ToString() : m_forcedChannelName;
```

Comment:
No semantic change: The only assignment to `m_forcedChannelName` is guaranteed to not be null: line 147 `m_forcedChannelName = m_config.GetString("forced_channel_name", String.Empty).ToLower();`